### PR TITLE
fix #9107 chore(cirrus): Push to dockerhub

### LINKED ISSUE #9107: Publish to Docker Hub
[https://github.com/mozilla/experimenter/blob/main/.circleci/config.yml#L350-L357](https://github.com/mozilla/experimenter/blob/main/.circleci/config.yml#L350-L357)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-3663)

### PR DESCRIPTION
**Pull request by yashikakhurana**
Because

- We want the cirrus image available 

This commit

- Pushed image to docker hub

### CODE REVIEW

File: .circleci/config.yml
Comment (yashikakhurana) [inline .circleci/config.yml:368]: @jaredlockhart we will be requiring a repo for cirrus to tag the latest, who can we reach out to do that?

**Comment by jeddai** [comment]
@yashikakhurana it looks like we use `mozilla/experimenter` as the repo https://hub.docker.com/r/mozilla/experimenter

Also if you log into CircleCI it looks like we have the ability to add environment variables. https://app.circleci.com/settings/project/github/mozilla/experimenter/environment-variables

**Comment by yashikakhurana** [comment]
blocking this PR- https://github.com/mozilla/experimenter/issues/9112

File: .circleci/config.yml
Comment (jaredlockhart) [inline .circleci/config.yml:373]: Couple things

1) We don't need to push anything other than deploy, the only reason we push the other stages for experimenter is so in theory your local build can use the remote cache, which worked for a while then stopped and I haven't had time to go back and figure it out, so just build and push deploy here

2) We should be able to reuse all the existing environment variables except for repo, in theory we should be able to push to `mozilla/cirrus`, ie those creds should have permission to that org.  Give that a shot.

**Review by jaredlockhart — DISMISSED** [review]

**Comment by yashikakhurana** [comment]
okay also I have added the environment variable- https://app.circleci.com/settings/project/github/mozilla/experimenter/environment-variables

File: .circleci/config.yml
Comment (jeddai) [inline .circleci/config.yml:367]: Should this be cirrus_build_prod maybe? It looks like that's what actually builds and tags as `cirrus:deploy` based on the Makefile. Also if we aren't pushing the test image and this job requires `Check Cirrus` I don't think we need to include the test here.
```suggestion
            make cirrus_build_prod
```

**Review by jaredlockhart — CHANGES_REQUESTED** [review]

File: .circleci/config.yml
Comment (jeddai) [inline .circleci/config.yml:616]: I'm thinking this may have been left in for testing but we'll def want this to be `only` before merging.
```suggestion
              only:
```

Comment (jeddai) [reply .circleci/config.yml:616]: Awesome just wanted to double check!

**Comment by jaredlockhart** [comment]
Closing in favour of https://github.com/mozilla/experimenter/pull/9118

**Pull request closed without merging**
